src/sheet_metal_conus.py

```diff
@@ -3,188 +3,338 @@
+            - center point, start angle, end angle, and a given style.
+            unitfactor scales all dimensions.
+        """
+        attribs = {'style' : simplestyle.formatStyle(style),
+        inkex.addNS('label','inkscape') : label,
+        inkex.addNS('cx','sodipodi')     :str(centerX * unitFactor),
+        inkex.addNS('cy','sodipodi')     :str(centerY * unitFactor),
+        inkex.addNS('rx','sodipodi')     :str(radiusX * unitFactor),
+        inkex.addNS('ry','sodipodi')     :str(radiusY * unitFactor),
+        inkex.addNS('start','sodipodi')  :str(startAngle),
+        inkex.addNS('end','sodipodi')    :str(endAngle),
+        inkex.addNS('open','sodipodi')   :'true',    #all ellipse sectors we will draw are open
+        inkex.addNS('type','sodipodi')   :'arc' }
+        ell = inkex.etree.SubElement(parent, inkex.addNS('path','svg'), attribs )
+
+    def drawLine(self, (x1, y1), (x2, y2), style, label, unitFactor, parent):
+        """ Draws a line from given point 1 to given point 2 with a given style.
+        """
+        line_attribs = {'style' : simplestyle.formatStyle(style), inkex.addNS('label','inkscape') : label,
+                        'd' : 'M %s,%s L %s,%s' % (x1*unitFactor, y1*unitFactor, x2*unitFactor, y2*unitFactor)}
+        line = inkex.etree.SubElement(parent, inkex.addNS('path','svg'), line_attribs )
+
+    def beVerbose(self, dictCone, unitFactor, parent):
+        """ Verbose output of calculated values. 
+            Can be used for debugging purposes or if calculated values needed.
         """
-        inkex.debug( "Base diameter: " + str(dictCone['diaBase'] ) )
-        inkex.debug( "Cut diameter: " + str(dictCone['diaCut'] ) )
-        inkex.debug( "Cone height: " + str(dictCone['heightCone'] ) )
-        inkex.debug( "Short radius: " + str(dictCone['shortRadius']) )
-        inkex.debug( "Long radius: " + str(dictCone['longRadius']) )
-        inkex.debug( "Angle of circle sector: " + str(dictCone['angle']) + " radians (= " + str(math.degrees(dictCone['angle'])) + " degrees)")
-        inkex.debug( "Chord length of base arc: " + str(dictCone['chordBase']) )
-        inkex.debug( "Chord length of cut arc: " + str(dictCone['chordCut']) )
-        inkex.debug( "ptA: " + str(dictCone['ptA']) )
-        inkex.debug( "ptB: " + str(dictCone['ptB']) )
-        inkex.debug( "ptC: " + str(dictCone['ptC']) )
-        inkex.debug( "ptD: " + str(dictCone['ptD']) )
-        inkex.debug( "DICT: " + str(dictCone) )
+        # unpack
+        base_dia = dictCone['diaBase']
+        cut_dia = dictCone['diaCut']
+        cone_height = dictCone['heightCone']
+        shortradius = dictCone['shortRadius']
+        longradius = dictCone['longRadius']
+        angle = dictCone['angle']
+        chord_base = dictCone['chordBase']
+        chord_cut = dictCone['chordCut']
+        ptA = dictCone['ptA']
+        ptB = dictCone['ptB']
+        ptC = dictCone['ptC']
+        ptD = dictCone['ptD']
+
+        # styles for markup
```

Comment:
Yeah - I took easy way out here(lot of work actually). Here's my reasoning :)
- chords. There are two lengths and one is long and one short. So I print them so outer one is always outside the inner one. Hoping user - even if confused can see that one is longer than other. Also reason why they are in their own color.

- angle. There are already two straight lines and word Angle and value lie between them. I could extend but when I look at some of the configs we get a lot of detail crowded together (also why diff colors).

So that's why I did it that way. Let me know how you want me to proceed...